### ISSUE TITLE
Weight file and cfg file type

### ISSUE THREAD

**Issue by ngingihy** [chuanenlin/drone-net#5]
Hi, I'm wondering what pre-trained weight you used for training to generate the yolo-drone-weights, was it yolov3-tiny/yolov2/voc? 

Also, did you do any pre-processing for the images before training?

**Comment by chuanenlin** [comment]
@ngingihy I used yolov3-tiny.cfg and weights from [darknet53](https://pjreddie.com/darknet/imagenet/#darknet53) (pre-trained on ImageNet). I didn't do any pre-processing before training.

**Comment by ngingihy** [comment]
So can i use your yolo-drone.weights as a pre-trained weight to train it
more on far away drones images, with the yolo-drone.cfg file? or you think
its better to start with  yolo-tiny and darknet53 weights?

**Comment by ngingihy** [comment]
So can i use your yolo-drone.weights as a pre-trained weight to train it more on far away drones images, with your yolo-drone.cfg file? or you think its better to start with  yolo-tiny and darknet53 weights?

**Comment by chuanenlin** [comment]
@ngingihy Yes, you can continue training from the yolo-drone.weights in the repo.
